### PR TITLE
[Bulk] Validation Fixes

### PR DESCRIPTION
**Pull request by AqMoh**
This PR fixes 23 issues

### CODE REVIEW

File: BingMaps/rest-services/routes/maneuver-types.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Maneuver Types | Microsoft Docs"
+description: "This article lists maneuver type values thar are returned the Routes in the maneuverType field in the HTTP response."
```
Comment (jmart1428) [inline BingMaps/rest-services/routes/maneuver-types.md:3]: typo
thar = that

File: BingMaps/rest-services/routes/snap-to-road-data.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Snap to Road Data | Microsoft Docs"
+description: "The response returned by the Snap to Road API contains either an SnapToRoadResponse resource, which contains the snapped point data, or an RouteProxyAsyncStatus resource. This article shows tables that provide descriptions of the SnapToRoadResponse resource fields."
```
Comment (jmart1428) [inline BingMaps/rest-services/routes/snap-to-road-data.md:3]: Too long per https://review.docs.microsoft.com/en-us/help/contribute/validation-ref/description-missing?branch=main

File: BingMaps/rest-services/timezone/index.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Time Zone| Microsoft Docs"
+description: "The Bing Maps Time Zone API makes it easy to find a time zone and daylight savings time for a location by query or latitude and longitude coordinates. This article provides a table of resources for this API."
```
Comment (jmart1428) [inline BingMaps/rest-services/timezone/index.md:3]: Too long per https://review.docs.microsoft.com/en-us/help/contribute/validation-ref/description-missing?branch=main

File: BingMaps/rest-services/timezone/list-time-zones.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "List Time Zones | Microsoft Docs"
+description: "The Time Zone API list operation returns a complete list of time zone information for either the Microsoft Windows or IANA Time Zone standard. This article provides API templates, parameters, and examples."
```
Comment (jmart1428) [inline BingMaps/rest-services/timezone/list-time-zones.md:3]: Too long per https://review.docs.microsoft.com/en-us/help/contribute/validation-ref/description-missing?branch=main

File: BingMaps/rest-services/traffic/traffic-incident-data.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Traffic Incident Data | Microsoft Docs"
+description: "The response returned by a Traffic URL contains one or more TrafficIncident resources. This article contains a table that describes the fields in the traffic incident resource. JSON and XML examples are shown."
```
Comment (jmart1428) [inline BingMaps/rest-services/traffic/traffic-incident-data.md:3]: Too long per https://review.docs.microsoft.com/en-us/help/contribute/validation-ref/description-missing?branch=main

**Comment by AqMoh** [comment]
**Global effort to fix build validation errors**
@stevemunk 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

File: BingMaps/rest-services/status-codes-and-error-handling.md [hunk 6]
```diff
@@ -17,7 +18,7 @@ ms.service: "bing-maps"
 
 Information on status codes and error handling.
 
-## Handling empty responses
+## Handling Empty Responses
```
Comment (stevemunk) [inline BingMaps/rest-services/status-codes-and-error-handling.md:21]: The capitalization in headers is wrong all over the place in the Bing Maps Docs See [here](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28718) "Microsoft style uses sentence-style capitalization. That means everything is lowercase except the first word of a sentence "

```suggestion
## Handling empty responses
```

File: BingMaps/rest-services/traffic/get-traffic-incidents.md [hunk 7]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Get Traffic Incidents | Microsoft Docs"
+description: "This article provides a URL template to request traffic incident information. Template parameters and examples are provided."
```
Comment (stevemunk) [inline BingMaps/rest-services/traffic/get-traffic-incidents.md:3]:
```suggestion
description: "This article describes how to get traffic incidents and provides a URL template to request traffic incident information. Template parameters and examples are provided."
```

File: BingMaps/rest-services/traffic/traffic-incident-data.md [hunk 8]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Traffic Incident Data | Microsoft Docs"
+description: "This article contains a table that describes the fields in the traffic incident resource. JSON and XML formats are shown."
```
Comment (stevemunk) [inline BingMaps/rest-services/traffic/traffic-incident-data.md:3]:
```suggestion
description: This article describes the Traffic Incident Data contained in TrafficIncident resources, returned by a Traffic URL, in a table that describes each field. Both JSON and XML formats are shown.
```

File: BingMaps/rest-services/using-the-rest-services-with-net.md [hunk 9]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Using the REST Services with .NET | Microsoft Docs"
+description: "This article describes how to interact with the Bing Maps REST Services APIs using .NET, covering making requests, implementing requents, and data contracts."
```
Comment (stevemunk) [inline BingMaps/rest-services/using-the-rest-services-with-net.md:3]:
```suggestion
description: "This article describes how to interact with the Bing Maps REST Services APIs using .NET, covering making use of sessions, making and implementing requests, and data contracts, includes sample code"
```

File: BingMaps/sdk-native/getting-started-ios/index.md [hunk 10]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Getting Started with iOS | Microsoft Docs"
+description: "This article is a tutorial that goes through creating an iOS app with a Bing Maps Native Control step-by-step."
```
Comment (stevemunk) [inline BingMaps/sdk-native/getting-started-ios/index.md:3]:
```suggestion
description: The getting started with iOS article is a tutorial that steps through creating an iOS app with a Bing Maps Native Control.
```

File: BingMaps/sdk-native/index.md [hunk 11]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Bing Maps SDK for Android and iOS | Microsoft Docs"
+description: "This article outlines Bing Maps SDK for Android and iOS, listing resources on how to get started, map control concepts, map control API reference, and map services API reference."
```
Comment (stevemunk) [inline BingMaps/sdk-native/index.md:3]:
```suggestion
description: The overview page for the Bing Maps SDK for Android and iOS section contains links to information about Bing Maps keys, license and terms of service and to the different topics covered such as getting started with Android and iOS, map control concepts and map control and services API references.
```

**Review by stevemunk — CHANGES_REQUESTED** [review]

**Comment by stevemunk** [comment]
@AqMoh Please let me know when you are ready for a final review. thank you

**Comment by AqMoh** [comment]
@stevemunk 
Requests have been incorporated. Please provide your review--thank you!

**Comment by stevemunk** [comment]
@AqMoh Some of the requested changes didn't get saved. I've unresolved a few of them. If you didn't make the requested changes because you don't agree with my feedback, that's fine, you might be right, but let's discuss.  Thanks!

**Comment by AqMoh** [comment]
@stevemunk 
Hi Steve! Your suggestions, on my end, are labeled as outdated and I am unable to commit/push them. Is this because the file has already been worked on by someone else?

**Comment by stevemunk** [comment]
@AqMoh Most likely. Have you synced your branch with main?

**Comment by jmart1428** [comment]
@stevemunk #please-close

**Pull request closed without merging**